src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -112,10 +112,15 @@ public void Initalize(IConfiguration configuration)
             }
             else
             {
+                HashSet<string> loadedFiles = new HashSet<string>();
```

Comment:
Should this be case-insensitive? (e.g. using `StringComparer.OrdinalIgnoreCase`)

Reply:
That certainly would not hurt 👍  Fixing